docs/docs/dev/vm-pod-lifecycle.md

```diff
@@ -51,7 +51,7 @@ Virtlet-specific annotation or an image that has Virtlet-specific prefix.
    `PullImage` CRI call to pull the image if it doesn't exist or if
    `imagePullPolicy: Always` is used.
  * If `PullImage` is invoked, Virtlet resolves the image location based on the
-   [image name translation configuration](https://github.com/Mirantis/virtlet/blob/master/docs/image-name-translation.md),
+   [image name translation configuration](https://github.com/hwchiu/virtlet/blob/master/docs/docs/reference/images.md#image-name-translation),
```

Comment:
For sure it's incorrect url because of `hwchiu` inside ;)

Reply:
Thanks, resolved.